zulip_bots/zulip_bots/bots/connect_four/test_connect_four.py

```diff
@@ -4,18 +4,20 @@
 from unittest.mock import MagicMock
 from zulip_bots.bots.connect_four.connect_four import *
 
+
 class TestConnectFourBot(BotTestCase):
     bot_name = 'connect_four'
 
-    def make_request_message(self, content, user='foo@example.com'):
+    def make_request_message(self, content, user='foo@example.com', user_name='foo'):
         message = dict(
             sender_email=user,
             content=content,
+            sender_full_name=user_name
         )
         return message
 
     # Function that serves similar purpose to BotTestCase.verify_dialog, but allows for multiple responses to be handled
-    def verify_response(self, request, expected_response, response_number, data=None, computer_move=None, user = 'foo@example.com'):
+    def verify_response(self, request, expected_response, response_number, data=None, user='foo@example.com'):
```

Comment:
Again, try to wrap the arguments at 80 characters.

And add mypy annotations for all of this code please. Otherwise the build will fail :)